org.lflang/src/org/lflang/LinguaFranca.xtext

```diff
@@ -389,6 +389,11 @@ terminal SL_COMMENT: ('//' | '#') !('\n'|'\r')* ('\r'? '\n')?;
 @Override
 terminal ML_COMMENT: ('/*' -> '*/') | ('\'\'\'' -> '\'\'\'');
 
+@Override
+terminal STRING:
+            '"' ( '\\' . /* 'b'|'t'|'n'|'f'|'r'|'u'|'"'|"'"|'\\' */ | !('\\'|'"') )* '"'
```

Comment:
Why are the escape values (like \n) commented out here? Do we really want to allow all possible characters after \? If so, then I think the commented out code should be removed. It took me quite a while to understand this rule.